### ISSUE TITLE
Disable console.log() messages

### ISSUE THREAD

**Issue by vinothox** [sonata-project/SonataAdminBundle#5278]
I see many console log messages in the browser generated by Sonata admin bundle.

![7kwb0](https://user-images.githubusercontent.com/30347402/47503978-f907f400-d888-11e8-9911-d2f64a7614e4.png)

These messages are generated from /vendor/sonata-project/admin-bundle/src/Resources/public/Admin.js by the code,

    /**
     * render log message
     * @param mixed
     */
    log: function() {
        var msg = '[Sonata.Admin] ' + Array.prototype.join.call(arguments,', ');
        if (window.console && window.console.log) {
            window.console.log(msg);
        } else if (window.opera && window.opera.postError) {
            window.opera.postError(msg);
        }
    },

I don't need these logs to be generated in the browser. Is there a way to control these from configuration file or something?

**Comment by OskarStark** [comment]
Does these logs occur on prod?

**Comment by vinothox** [comment]
Yes. Both dev and prod environment.

**Comment by bgaleotti** [comment]
You can customize the layout and override that function:

```jinja
{% extends 'SonataAdminBundle::standard_layout.html.twig' %}

{% block javascripts %}
  {{ parent() }}
  <script>
    if ('undefined' !== typeof window.Admin) {
      window.Admin.log = function() {}
    }
  </script>
{% endblock %}
```

**Comment by vinothox** [comment]
@bgaleotti Thanks, it works.

**Comment by vinothox** [comment]
But logging console messages even in production environment is the actual implementation or is it a bug?

**Comment by core23** [comment]
IMHO we should switch the actual behaviour. So that we have a logger function that does nothing, but you can override/enable it if you want

**Comment by vinothox** [comment]
Yes, I have overrided the standard_layout to disable it. Thanks for the support. Can I close this issue now?

**Comment by OskarStark** [comment]
No please keep it to open until we have a better solution

**Comment by vinothox** [comment]
Sure
